src/hidimstat/knockoff_aggregation.py

```diff
@@ -87,10 +87,10 @@ def knockoff_aggregation(
     adaptive_aggregation : bool, default=False
         Whether to apply the adaptive version of the quantile aggregation method
         as in :footcite:t:`Meinshausen_2008`.
-    gamma: float, default=0.5
+    gamma : float, default=0.5
         The percentile value used for aggregation.
-    gamma_min : float, default=0.05
```

Comment:
you're now implicitly setting gamma_min to its default, which is .05. Why not keep the parameter accessible.
Note that `gamma_min` has probably more impact on the result than `n_grid_gamma`

Reply:
OK, but in this case, I don't see the advantage of having one function for quantile_aggregation.
I propose to remove the function `quantile_aggregation` and make the functions `_fixed_quantile_aggregation` and `_adaptive_quantile_aggregation` public.

Reply:
It's what I did.
I remove `gamma_min` and replaced it with `gamma`.
Sorry, I forgot to modify the docstring associated with it.

Reply:
@bthirion

Reply:
@bthirion